test/elixir_sense/core/introspection_test.exs

```diff
@@ -41,6 +41,7 @@ defmodule ElixirSense.Core.IntrospectionTest do
            """
   end
 
+  @tag requires_elixir_1_9: true
```

Comment:
Most of those tags are incorrect as the code forks fine on earlier elixir/otp combos. The tests unfortunately need to assert on doc strings and those are release dependant. A better name or a comment would make things clear

Reply:
I understand the ambiguity and I am open to suggestions of better naming.

A third, more laborious, route is to make the tests actually compatible with all versions or, even better, version-agnostic.